Dockerfile

```diff
@@ -33,14 +33,18 @@ RUN easy_install3 pip
 RUN easy_install pip
 RUN pip3 install -U virtualenv auxlib
 RUN pip2 install -U virtualenv auxlib
-RUN curl -O https://repo.continuum.io/miniconda/Miniconda2-4.2.11-Linux-x86_64.sh
-RUN bash Miniconda2-4.2.11-Linux-x86_64.sh -b -p /usr/local/miniconda/
-RUN ln -s /usr/local/miniconda/bin/conda /usr/local/bin/conda
 
 # UID and GID from readthedocs/user
 RUN groupadd --gid 205 docs
 RUN useradd -m --uid 1005 --gid 205 docs
 
 USER docs
 
+# Install miniconda as docs user
+WORKDIR /home/docs
+RUN curl -O https://repo.continuum.io/miniconda/Miniconda2-4.2.11-Linux-x86_64.sh
+RUN bash Miniconda2-4.2.11-Linux-x86_64.sh -b -p /home/docs/miniconda2/
+RUN mkdir /home/docs/bin
+RUN ln -s /home/docs/miniconda2/bin/conda /home/docs/bin/conda
```

Comment:
Instead of relying on the shell to pick this up, i think we should implement a pattern similar to the `latest` image:

https://github.com/rtfd/readthedocs-docker-images/blob/master/Dockerfile#L52-L53

That is, install to `~docs/.conda` and add this to `PATH` explicitly.

I really don't know if this works, but will likely be the pattern we stick to as we for other tools that expect binaries locally for users.